lib/core/src/lib/notifications/models/notification.model.ts

```diff
@@ -39,4 +39,5 @@ export interface NotificationModel {
     icon?: string;
     clickCallBack?: any;
     args?: any;
+    read: boolean;
```

Comment:
Is not that breaking change? What if customer uses ADF and they use NotificationModel in their code? They would need to apply 'read' to every place in their project as it is required field. 

Please mark that field as optional: 
read?: boolean;

So then it won't break their code.

Reply:
I already took care of 'read' by updating all methods in notification.factory.ts https://github.com/Alfresco/alfresco-ng2-components/pull/9011/files#diff-591c0e2318e479867119518c85813996a2cfe6b5dff58d0160048dbccd6472c6R35 and in ADW while configuring recursive notifications https://github.com/Alfresco/alfresco-applications/pull/412/files#diff-cf6c3613c33d6b7ae76641923342b50ab8c359e4f8fe3ba2dde6d98f931333eeR37.

But I have made 'read' optional to avoid any unforeseen breaking changes.